packages/contracts/contracts/RisingTide/RisingTide.sol

```diff
@@ -38,7 +38,7 @@ abstract contract RisingTide {
 
     /// Min gas required to run one more cap validation iteration
     /// TODO tweak this value
-    uint256 public constant CAP_VALIDATION_GAS_LIMIT = 50000;
+    uint256 public constant CAP_VALIDATION_GAS_LIMIT = 100000;
```

Comment:
Why did this double?

Reply:
because I was running into errors when running the specs that simulate 1000 investors. I guess the previous value was just not correct at all